### PR TITLE
Update extending.rst

### PR DESCRIPTION
**Pull request by andytwoods**
I thought this might save someone a little time.

### CODE REVIEW

File: docs/extending.rst [hunk 1]
```diff
@@ -113,3 +113,20 @@ You will need to customize the following templates:
...
+
+
+Modifying comments with code
+====================
+Here's an example of how to access the underlying model storing your comments
+
+    from django_comments_xtd.models import XtdComment
+    from django.contrib.contenttypes.models import ContentType
+    
+    def make_comments_nonpublic(model_instance):
```
Comment (danirus) [inline docs/extending.rst]: This function doesn't make comments non-public. It does only update the `followup` but exclude some email address from the update. Why the name of the function doesn't match what it does? Could you elaborate the goal of this new section of the documentation? I'm a bit lost here.

File: docs/extending.rst [hunk 2]
```diff
@@ -113,3 +113,20 @@ You will need to customize the following templates:
...
+        content_type = ContentType.objects.get_for_model(model_instance)
+        excluded = ["joe.bloggs@example.com", "bla@bla.com"]
+
+        XtdComment.objects\
+            .filter(content_type=content_type, object_pk=job.pk)\
```
Comment (danirus) [inline docs/extending.rst]: Where does `job` comes from?

File: docs/extending.rst (hunk 1 above)
Comment (andytwoods) [reply docs/extending.rst]: you are totally right. I've renamed (was relevant for my usage)

File: docs/extending.rst (hunk 2 above)
Comment (andytwoods) [reply docs/extending.rst]: I've removed this silliness.

**Comment by danirus** [comment]
@andytwoods, thanks for your contribution!

**Pull request merged**